### PR TITLE
Wikivoyage by Kiwix 2.5.2 UWP

### PR DESCRIPTION
**Pull request by Jaifroid**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/110980&drop=dogfoodAlpha

### CODE REVIEW

File: manifests/k/Kiwix/Wikivoyage/2.5.2.0/Kiwix.Wikivoyage.installer.yaml
Comment (BrandonWanHuanSheng) [inline manifests/k/Kiwix/Wikivoyage/2.5.2.0/Kiwix.Wikivoyage.installer.yaml:11]:
```suggestion
https://github.com/kiwix/kiwix-js-windows/releases/download/v2.5.2-Wikivoyage/KiwixWebAppWikivoyage_2.5.2.0_AnyCPU.appxbundle
```

File: manifests/k/Kiwix/Wikivoyage/2.5.2.0/Kiwix.Wikivoyage.installer.yaml
Comment (BrandonWanHuanSheng) [inline manifests/k/Kiwix/Wikivoyage/2.5.2.0/Kiwix.Wikivoyage.installer.yaml:12]:
```suggestion
  InstallerSha256: 29AD1484259A2DD22A11AA9966299846BEE8587CF007484F92C12F0D40F74132
```

**Review by BrandonWanHuanSheng — CHANGES_REQUESTED** [review]
It look like that file is not found on the server. https://github.com/kiwix/kiwix-js-windows/releases/download/v2.5.2-Wikivoyage/Wikivoyage.by.Kiwix.2.5.2-E.appx got 404.

**Comment by BrandonWanHuanSheng** [comment]
[Policy]: Reset-Feedback

**Comment by BrandonWanHuanSheng** [comment]
[Policy] feedback reset

**Review by BrandonWanHuanSheng — COMMENTED** [review]
Sorry because github is hiding the setup package

**Review by BrandonWanHuanSheng — APPROVED** [review]
Sorry, I have deleted by suggestion comment because it is marked as outdated due to the setup file it might be hide on the release file on the package. when downloading from github. Becareful, could be some link was hidden when downloading from the release notes. github.com

**Comment by BrandonWanHuanSheng** [comment]
I get confused when github.com hide its setup installer.

**Comment by Jaifroid** [comment]
@BrandonWanHuanSheng Thank you. Is there anything I need to change? Yes, some assets are hidden until you press "Show all 15 assets". It seems the PR is just waiting for mod approval now?

**Comment by BrandonWanHuanSheng** [comment]
> @BrandonWanHuanSheng Thank you. Is there anything I need to change? Yes, some assets are hidden until you press "Show all 15 assets". It seems the PR is just waiting for mod approval now?

No, need.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**
